practice/tic-tac-toe/coords_board.py

```diff
@@ -0,0 +1,106 @@
+"""This module performs the functions for Tic-Tac-Toe (TTT) program of placing a token on the board, determining the
+winner of a game, and returning a str version of the board, using tuples data type.
+"""
+from operator import itemgetter
+
+WINNING_COMBINATIONS = [[(0, 0), (1, 0), (2, 0)], [(0, 0), (0, 1), (0, 2)], [(1, 0), (1, 1), (1, 2)], [(0, 1), (1, 1),
+                                                                                                       (1, 2)],
+                        [(2, 0), (2, 1), (2, 2)], [(0, 2), (1, 2), (2, 2)], [(0, 0), (1, 1), (2, 2)], [(2, 0), (1, 1),
+                                                                                                       (0, 2)]]
```

Comment:
If there are patterns in literals or in code, have the computer generate it, rather than you generating it. It's easy to make a mistake incrementing one on some number. Using `range()` and concatenating some lists makes this easy.